speech/export-to-storage.v1p1beta1.js

```diff
@@ -0,0 +1,104 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+function main(
+  inputUri,
+  outputStorageUri,
+  encoding,
+  sampleRateHertz,
+  languageCode,
+  bucketName,
+  objectName
+) {
+  // [START speech_export_to_gcs]
```

Comment:
This sample is not (yet) used in our docs.

@meredithslota I presume you're planning to add it to `cloud.google.com` somewhere?

Reply:
@ace-n actually I was able to find this sample in the docs by searching on the path rather than region tag.

+1 about tests. There are a couple matching test files we could add to this PR from https://github.com/googleapis/nodejs-speech/tree/main/samples/system-test.